models/base/manifest/snowplow_web_base_sessions_lifecycle_manifest.sql

```diff
@@ -6,25 +6,41 @@
     full_refresh=false,
     schema=var("snowplow__manifest_custom_schema"),
     sort='start_tstamp',
-    dist='session_id'
+    dist='session_id',
+    partition_by = {
+      "field": "start_tstamp",
+      "data_type": "timestamp",
+      "granularity": "day"
+    },
+    cluster_by=["session_id"]
   ) 
 }}
 
-with sessions_this_run as (
+-- Edge case 1: Rare case with multiple domain_userid per session.
```

Comment:
nit: On first reading this comment was confusing, but I now see that it's because the below comments refer to the same edge case. Not sure how, but if we can think of a way to make that clearer here that'd be nice.

Reply:
Fair point. Good to get your fresh pair of eyes perspective on that. Will see what I can do